.github/ISSUE_TEMPLATE/operational-issues.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Operational issues
+about: Website and hosting-related issues
+title: ''
+labels: site
+assignees: ''
+
+---
+
+<!-- This issue type is mostly for contributors and developers. If you are seeing an issue with the website and you’re not involved in this project, you probably want a different issue type.
+
+Also, note that the site is available in at least two places: https://docs.perl6.org/ and https://perl6docs.github.io/ (and the latter will usually be more up to date).
```

Comment:
What is https://perl6docs.github.io it's the first time I see it.

Reply:
It’s referenced in several places, such as:

https://github.com/perl6/doc/blob/58b9ef05015db6b58d7300e967eb31fb9020c0f6/README.md#L5-L9